src/parser/rogue/outlaw/modules/spells/shadowlands/legendaries/GuileCharm.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Analyzer from 'parser/core/Analyzer';
+import Abilities from 'parser/core/modules/Abilities';
+import SPELLS from 'common/SPELLS';
+import Statistic from 'interface/statistics/Statistic';
+import { formatPercentage } from 'common/format';
+import BoringSpellValue from 'interface/statistics/components/BoringSpellValue';
+import SpellLink from 'common/SpellLink';
+
+class GuileCharm extends Analyzer {
+  static dependencies = {
+    abilities: Abilities,
+  };
+
+  deepInsightUptime: number = 0;
+  moderateInsightUptime: number = 0;
+  shallowInsightUptime: number = 0;
+  protected abilities!: Abilities;
+
+  constructor(options: any) {
+    super(options);
+    this.active = this.selectedCombatant.hasLegendaryByBonusID(SPELLS.GUILE_CHARM.bonusID);
+  }
+
+  get percentUptime() {
+    this.shallowInsightUptime = this.selectedCombatant.getBuffUptime(SPELLS.SHALLOW_INSIGHT_BUFF.id) / this.owner.fightDuration;
+    this.moderateInsightUptime = this.selectedCombatant.getBuffUptime(SPELLS.MODERATE_INSIGHT_BUFF.id) / this.owner.fightDuration;
+    this.deepInsightUptime = this.selectedCombatant.getBuffUptime(SPELLS.DEEP_INSIGHT_BUFF.id) / this.owner.fightDuration;
+    return {
+      shallowInsight: this.shallowInsightUptime,
+      moderateInsight: this.moderateInsightUptime,
+      deepInsight: this.deepInsightUptime,
+    };
+  }
+
+  statistic() {
+    return (
+      <>
+        <Statistic size="flexible" tooltip="This measures how long each buff from this legendary is active.">
+          <div className="pad">
```

Comment:
not sure what this div/label is accomplishing. Maybe a screenshot of the stat box would help.

Reply:
![image](https://user-images.githubusercontent.com/23483002/95269719-a9196200-07ff-11eb-9a74-ddb30c33403c.png)
This adds padding so the text isn't pushed up against the corner with no padding